### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by steamfoundry**
Just wanted to make all the code blocks match in the README.  
I also shorted the paths of the Windows examples to make them simpler to read.

### CODE REVIEW

**Comment by steamfoundry** [comment]
@ladyada, @PaintYourDragon or @kattni If one of you has time could you review this please?

File: README.md [hunk 1]
```diff
@@ -5,21 +5,15 @@ examples:
 
 RP2040:
 
-```
-python3 parser.py "Adafruit Feather RP2040.fzpz" C:\Users\ladyada\Dropbox\micropython\circuitpython\ports\raspberrypi\boards\adafruit_feather_rp2040\pins.c rp2040pins.csv
-```
+```python3 parser.py "Adafruit Feather RP2040.fzpz" ~/Git/circuitpython/ports/raspberrypi/boards/adafruit_feather_rp2040/pins.c rp2040pins.csv```
```
Comment (kattni) [inline README.md:8]: Please make the path more generalised. Consider the following.
```suggestion
```python3 parser.py "Adafruit Feather RP2040.fzpz" path/to/circuitpython/ports/raspberrypi/boards/adafruit_feather_rp2040/pins.c rp2040pins.csv```
```

File: README.md [hunk 2]
```diff
@@ -5,21 +5,15 @@ examples:
...
 
-```
-python3 parser.py "Adafruit ItsyBitsy RP2040.fzpz" C:\Users\ladyada\Dropbox\micropython\circuitpython\ports\raspberrypi\boards\adafruit_feather_rp2040\pins.c rp2040pins.csv
-```
+```python3 parser.py "Adafruit ItsyBitsy RP2040.fzpz" ~/Git/circuitpython/ports/raspberrypi/boards/adafruit_feather_rp2040/pins.c rp2040pins.csv```
```
Comment (kattni) [inline README.md:10]: Same suggestion as above. Also this path needs `feather` to be updated to `itsybitsy`.
```suggestion
```python3 parser.py "Adafruit ItsyBitsy RP2040.fzpz" _~/Git/circuitpython/ports/raspberrypi/boards/adafruit_itsybitsy_rp2040/pins.c rp2040pins.csv```
```

File: README.md [hunk 3]
```diff
@@ -5,21 +5,15 @@ examples:
...
 
-```
-python3 parser.py "Adafruit QT Py RP2040.fzpz" C:\Users\ladyada\Dropbox\micropython\circuitpython\ports\raspberrypi\boards\adafruit_qtpy_rp2040\pins.c rp2040pins.csv
-```
+```python3 parser.py "Adafruit QT Py RP2040.fzpz" C:\Users\ladyada\Git\circuitpython\ports\raspberrypi\boards\adafruit_qtpy_rp2040\pins.c rp2040pins.csv```
```
Comment (kattni) [inline README.md:12]: Same.

File: README.md [hunk 4]
```diff
@@ -5,21 +5,15 @@ examples:
...
 
 ESP32-S2:
 
-```python3 parser.py "Adafruit Metro ESP32-S2.fzpz" C:\Users\ladyada\Dropbox\micropython\circuitpython\ports\esp32s2\boards\adafruit_metro_esp32s2\pins.c esp32s2pins.csv -s "^D([0-9])" "IO\1"```
+```python3 parser.py "Adafruit Metro ESP32-S2.fzpz" C:\Users\ladyada\Git\circuitpython\ports\esp32s2\boards\adafruit_metro_esp32s2\pins.c esp32s2pins.csv -s "^D([0-9])" "IO\1"```
```
Comment (kattni) [inline README.md:16]: Same.

**Review by kattni — CHANGES_REQUESTED** [review]
Thanks for fixing this up! It's something I meant to do. I'd like to see the paths more generalised. I made a suggestion below.

**Review by kattni — APPROVED** [review]
Thanks for the fix!

File: README.md (hunk 1 above)
Comment (steamfoundry) [reply README.md]: That looks better.   They all match now. This meant there was a duplicate entry for the Metro ESP32-S2.  Would you rather keep one with an example Windows file system or have them all line up?

File: README.md (hunk 2 above)
Comment (steamfoundry) [reply README.md]: Fixed

**Comment by steamfoundry** [comment]
Thanks for the quick review @kattni!

File: README.md (hunk 1 above)
Comment (kattni) [reply README.md]: I missed this question in my review. I'd rather they all match up. If you want to remove the duplicate, that would be great.

**Pull request merged**